src/wallet/wallet.h

```diff
@@ -782,9 +782,12 @@ class CWallet final : public CCryptoKeyStore, public CValidationInterface
     unsigned int nMasterKeyMaxID;
 
     /** Construct wallet with specified name and database implementation. */
-    CWallet(std::string name, std::unique_ptr<CWalletDBWrapper> dbw) : m_name(std::move(name)), dbw(std::move(dbw))
+    CWallet(std::string name, std::unique_ptr<CWalletDBWrapper> dbw) : fAbortRescan(false), fScanningWallet(false),
+        pwalletdbEncryption(nullptr), nWalletVersion(FEATURE_BASE), nWalletMaxVersion(FEATURE_BASE), nNextResend(0),
+        nLastResend(0), fBroadcastTransactions(false), m_max_keypool_index(0), nTimeFirstKey(0),
+        m_name(std::move(name)), dbw(std::move(dbw)), m_last_block_processed(nullptr), nMasterKeyMaxID(0),
+        nOrderPosNext(0), nAccountingEntryNumber(0), nRelockTime(0)
```

Comment:
Shoulnd't that be initialized where it was defined according to the developer notes? https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#c-data-structures